### ISSUE TITLE
executeDocumentSymbolProvider

### ISSUE THREAD

**Issue by majeric** [OmniSharp/omnisharp-vscode#2192]
## Environment data
`dotnet --info` output:
```
.NET Command Line Tools (2.1.4)

Product Information:
 Version:            2.1.4
 Commit SHA-1 hash:  5e8add2190

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.16299
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.4\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.5
  Build    : 17373eb129b3b05aa18ece963f8795d65ef8ea54
```
VS Code version: 1.22.2
C# Extension version: 1.14.0

## Steps to reproduce
When creating an extension, omnisharp-vscode doesn't seem to produce a symbol array. (Where as a lot of other extensions do support this.) despite the fact that omnisharp-vscode does have a DocumentSymbolProvider

```TypeScript
 var active = vscode.window.activeTextEditor.document.uri;
 vscode.commands.executeCommand('vscode.executeDocumentSymbolProvider', active)
```

## Expected  behavior
Getting an array back from the executeDocumentSymbolProvider of the C# document that's open and in focus.

## Actual behavior

Empty array.

**Comment by rchande** [comment]
@majeric I would expect this to work. Share your OmniSharp log?

**Comment by majeric** [comment]
The symbol treeview extension should be listing the C# symbols based on the defined DocumentSymbolProvider. It works for a lot of other languages. For some reason, I can’t find out why the symbols return from that execute call.

https://marketplace.visualstudio.com/items?itemName=igbenic.symbolstreeview

**Comment by majeric** [comment]
I should add that the cmd-shift-o symbol outline list does work well in Visual Studio Code so clearly the DocumentSymbolProvider is functioning in some capacity. It's just odd to me that the Symbols Treeview wouldn't work as well (Symbol summary windows are super helpful in getting a bird's eye view of large classes and large documents). I did download the source and ran it through a debugger and the 'vscode.executeDocumentSymbolProvider' was returning an empty list for C# documents (despite the fact that omnisharp does have a DocumentSymbolProvider. ) However, the abstraction is significant enough that I hoped that the original developers (you guys) would have quicker insight as to why it wasn't producing a symbol list.

I've recently switched to Visual Studio Code for Unity and I'm enjoying it but I do miss some MonoDevelop features. (Symbol summary window being one of them)
